### PR TITLE
fix: fix the path of swagger

### LINKED ISSUE #577: Bug: OpenAPI Document endpoint is wrong, now docs/images/index.html, expect docs/index.html
### What would you like to be added?

The OpenAPI Document endpoint is wrong, now `docs/images/index.html`, expect `docs/index.html`

### Why is this needed?

yeah

### PR DESCRIPTION
**Pull request by regend**
## What type of PR is this?
update http-swagger to v2 to solve the path docs/images/index.html -> docs/index.html
## What this PR does / why we need it:

#https://github.com/swaggo/http-swagger/issues/10

## Which issue(s) this PR fixes:

Fixes #577

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -64,6 +63,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/swaggo/files/v2 v2.0.0 // indirect
```
Comment (elliotxx) [inline go.mod:66]: This `indirect require` should be merged into the `require block` below. You can clip-paste it to below, then execute `go mod tidy`.

**Comment by elliotxx** [comment]
@regend Good Job! Have you tested the content in this PR?

**Comment by elliotxx** [comment]
Ohh, CI is failed. It seems unrelated to your PR, let me fix it in the main branch.

**Comment by regend** [comment]
> @regend Good Job! Have you tested the content in this PR?

@elliotxx yes.  I have tried both docs/index.html and docs/images/index.html. It both work.

**Review by elliotxx — APPROVED** [review]
lgtm

**Pull request merged**